docs/src/reference/asciidoc/core/intro/requirements.adoc

```diff
@@ -87,6 +87,8 @@ Hive version {hv-v}
 
 Pig 0.10.0 or higher. We recommend using the latest release of Pig (currently {pg-v}).
 
+Note that support for Storm in {eh} is now deprecated.
+
```

Comment:
I'd move this to a `deprecated` admonition so it's a little more visible.
```suggestion
deprecated::[8.0,Support for Apache Pig in {eh} is deprecated.]

Pig 0.10.0 or higher. We recommend using the latest release of Pig (currently {pg-v}).
```

Here's how it looks:

<img width="771" alt="Screen Shot 2022-01-05 at 4 32 09 PM" src="https://user-images.githubusercontent.com/40268737/148292479-b46c77de-ada9-4da1-972d-82a0af01fa4e.PNG">

As a note, the current text refers to "Storm," but I think it should be "Pig."